source/upgrade.txt

```diff
@@ -47,9 +47,36 @@ dialog.
    :figwidth: 600px
    :alt: Privacy settings
 
-|compass| does not install any updates without your explicit permission.
-Instead, |compass-short| prompts you to upgrade when a new version is
-available.
+If you enable auto updates and have not downgraded |compass|,
+|compass-short| automatically updates the application four hours after
+MongoDB releases new versions. The update does not require your explicit
```

Comment:
I wouldn't mention a specific time frame here (and the 4 hours thing is not really accurate anyway)

Reply:
Thanks for this clarification! I removed the mentions of 4 hours